code/components/extra-natives-five/src/GamePrimitives_Outlines.cpp

```diff
@@ -604,6 +608,21 @@ static InitFunction initFunctionScriptBind([]()
 		}
 	});
 
+	fx::ScriptEngine::RegisterNativeHandler("SET_ENTITY_DRAW_OUTLINE_RENDER_TECHNIQUE", [](fx::ScriptContext& context)
+	{
+		std::string techniqueGroupId = context.GetArgument<const char*>(0);
```

Comment:
```suggestion
		std::string techniqueGroupId = context.CheckArgument<const char*>(0);
```
Using `CheckArgument` here will make this error if index `0` is null

Reply:
Good point! I thought the type checking in *natives* would handle that. I'll do some tests and fix it in upcoming commits